dora/tests/src/test/java/alluxio/client/fs/io/DoraPositionReaderIntegrationTest.java

```diff
@@ -79,7 +76,6 @@ public static Collection<Object[]> data() {
           .setProperty(PropertyKey.USER_STREAMING_READER_CHUNK_SIZE_BYTES, Constants.KB)
           .setProperty(PropertyKey.WORKER_BLOCK_STORE_TYPE, "PAGE")
           .setProperty(PropertyKey.WORKER_PAGE_STORE_PAGE_SIZE, Constants.KB)
-          .setProperty(PropertyKey.WORKER_PAGE_STORE_DIRS, PAGE_STORE)
```

Comment:
why these changes?

Reply:
because i updated the LocalAlluxioCluster to assigin the page store dirs automatically so there is no need to set the dir manually.